scripts/jenkins/Jenkinsfile

```diff
@@ -156,9 +156,19 @@ pipeline {
             }
         }
         success {
-            // If all tests passed, clean everything up
-            sh "vagrant destroy -f ${params.Box}"
-            cleanWs()
+            script {
+                if (env.BRANCH_NAME.startsWith("release")) {
+                    // Don't clean up this workspace.  We want to leave it up for acceptance testing
+                    def VERS = sh(returnStdout: true, script: "vagrant ssh default -c 'cd hoot; ./scripts/git/GitVersion.sh'")
```

Comment:
Does `default` need to be replaced with `${params.Box}`?

Reply:
Yep.  Nice catch.  Will make the change.